templates/hooks/hosted_zone/sdk_file_end.go.tpl

```diff
@@ -0,0 +1,23 @@
+{{ $CRD := .CRD }}
+{{ $SDKAPI := .SDKAPI }}
+
+{{/* Generate helper methods for HostedZone */}}
+{{- range $specFieldName, $specField := $CRD.Config.Resources.HostedZone.Fields }}
+{{- if $specField.From }}
+{{- $operationName := $specField.From.Operation }}
+{{- $operation := (index $SDKAPI.API.Operations $operationName) -}}
+{{- range $hostedZoneRefName, $hostedZoneMemberRefs := $operation.InputRef.Shape.MemberRefs -}}
+{{- if eq $hostedZoneRefName "AddTags" }}
```

Comment:
If i get this well, `AddTags` is a field part of of `CreateHostedZone` operation?

Reply:
You are right. To be specific, `Tags` is a field in `HostedZone` resource. `operationName` is `ChangeTagsForResource`. `AddTags` is memberRef in that operation.